src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -334,6 +336,24 @@ void CascadiaSettings::_ValidateProfilesExist()
     }
 }
 
+// Method Description:
+// - Checks if the profiles contain invalid environment variable values
+//   profiles at all, we'll throw an error if there aren't any profiles.
+void CascadiaSettings::_ValidateProfileEnvironmentVariables()

```

Comment:
This does not throw an error if there aren't any profiles.

Reply:
sorry, copy pasta